src/job-posting/dto/job-response.dto.ts

```
@@ -0,0 +1,73 @@
+  }
+}
+
+export class PaginatedJobResponseDto {
+  data: JobResponseDto[]
+  total: number
+  page: number
+  limit: number
+  totalPages: number
+
+  constructor(jobs: Job[], total: number, page: number, limit: number) {
+    this.data = jobs.map((job) => new JobResponseDto(job))
```

ESLint

[error] Insert `;`

(prettier/prettier)